docs/development/Create-a-new-StartLeft-Processor.md

```diff
@@ -0,0 +1,460 @@
+# Create a new StartLeft Processor (SLP)
+As you can see in the [architecture page](Architecture.md) StartLeft Processors (SLPs) are the special kind of 
+modules
+where the conversion into OTM logic is actually implemented. All of them are based in the same interfaces, defined in the 
+`slp_base` module. So, if you want to create a new SLP, you simply have to:
+
+* Create your module extending `slp_base`.
+* Implement the parsing logic in your module.
+* Configure StartLeft to use your module.
+
+Below you have a guided tutorial for easily performing this steps and creating a new SLP for a very basic invented 
+input format that you can use as a base for building your own.
+
+## Defining a sample input format (for demo purposes)
+
+---
+> **Note**: The SLPs are classified in IaC, Diagram or External Threat Model (ETM). However, it is only
+> a logical division and the instructions defined in this tutorial are valid for creating SLPs of any kind.
+
+To guide this tutorial, let's imagine a fictitious Infrastructure as Code format, that we will call _My Awesome Infrastructure
+Source (MAIS)_. A sample file of this format could be something like:
+```json
+{
+  "resources": [
+    {
+      "id": 1,
+      "name": "My Server",
+      "type": "server"
+    }
+  ]
+}
+```
+
+We need to define a format for the mappings between the source file and OTM. An example of this mapping file could be:
```

Comment:
Here we should probably link to the mapping file docs, because the user could reach this point of the guide without having any context about how the mapping files work or what they do.

Reply:
The point is that each SLP may have its own mapping file format and parsing strategy, there is nothing generic here. Actually, this section is just to create an arbitrary and trivial mapping example. Have you in mind some section of the docs that could be useful to link here?

Reply:
Maybe in the future we could have a section with a general explanation of mapping files and why they are needed, but I agree that this should not block the task because the users can refer to each specific format to learn about the different mapping files.